reconcile/utils/terrascript_client.py

```diff
@@ -1192,13 +1193,13 @@ def generate_random_password(string_length=20):
                        for i in range(string_length))
 
     def populate_tf_resource_s3(self, resource, namespace_info):
-        account, identifier, common_values, \
-            output_prefix, output_resource_name = \
+        account, identifier, common_values, output_prefix, \
+            output_resource_name, annotations = \
```

Comment:
OMG...

Can you please cut me a task to simplify all this duplicated code?

Reply:
https://issues.redhat.com/browse/APPSRE-3505